### PR TITLE
[CSL-2234] Updates Quizzes API with new response & request properties 

### PR DESCRIPTION
**Pull request by Mudaafi**

### CODE REVIEW

File: src/Constructorio_NET.Tests/models/Quizzes/QuizRequestTest.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public void GetRequestParameters()
             {
                 UserInfo = this.UserInfo,
                 Answers = this.Answers,
-                VersionId = this.VersionId,
```
Comment (stanlp1) [inline src/Constructorio_NET.Tests/models/Quizzes/QuizRequestTest.cs:44]: Let's also add sessionId to this test.

**Review by stanlp1 — APPROVED** [review]
Small comment, other than that this is looking good to me.

File: src/constructor.io/models/Quizzes/QuizRequest.cs [hunk 2]
```diff
@@ -24,7 +24,12 @@ public class QuizRequest
         /// <summary>
         /// Gets or sets version identifier for the quiz.
```
Comment (stanlp1) [inline src/constructor.io/models/Quizzes/QuizRequest.cs:25]: Should we make this more verbose with an explanation about when to use this?

https://github.com/Constructor-io/constructorio-client-javascript/pull/245#discussion_r1150842443

Comment (Mudaafi) [reply src/constructor.io/models/Quizzes/QuizRequest.cs]: Totally, I missed out what we did for client-js but it's definitely nice to keep things consistent across SDKs.

**Pull request merged**